templates/admins/edit-profile-admin.html

```diff
@@ -2,29 +2,33 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta name="description" content="A page for editing admin information" />
-  <link rel="stylesheet" href="../../dist/output.css" />
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Don't think this is necessary? `<meta charset="UTF-8" />` and `<meta charset="UTF-8">` are the same? Removing the `/` does not make a difference.